library/ssl_srv.c

```diff
@@ -35,6 +35,7 @@
 #define mbedtls_free      free
 #endif
 
+#include "mbedtls/error.h"
```

Comment:
Should we keep includes in ABC order? Here and all other places in this PR where we add an include for error.h

Reply:
If they were already in alphabetical order, I think it would worth considering keeping it that way. But as far as I can tell they are not and therefore I think this should be out of scope for the PR.

Reply:
They mostly are in order, with the module header often included first to avoid buried dependencies.

Reply:
I am sorry I can't see it. Could you please explain what you mean using this instance as an example? I mean where would you like to put this include in this file?

Reply:
I have pushed a new commit with the reordering, I hope this is what you meant.